sources/platform/monitoring/index.md

```diff
@@ -1,94 +1,108 @@
 
-## How can I set up monitoring? {#how-can-i-set-up-monitoring}
+![Metric condition configuration](./images/metric-options.png)
 
-We currently have video tutorials for setting up monitoring and adding data validation.
+You can get notified by email, Slack, or in the Apify Console. If you use Slack, we suggest using Slack notifications instead of email, because they are more reliable and you can also get notified quicker.
 
-* [Set up monitoring](https://www.youtube.com/watch?v=bLVk5dkFA7Y).
+1. **Email** - You can set up one or more emails to receive alert notifications, to do that you just have to separate them by commas. You can also disable email notifications if you don't want to receive them.
+2. **Slack** - To set up Slack notifications you first need to connect your Slack workspace to Apify. To do that, go to your [account integration settings](https://console.apify.com/account/integrations) and click on the **+ Add workspace** button in the Slack section. Once you have your workspace connected, you can choose the workspace when setting up alert notifications and then pick a channel where you want the notifications to be delivered to.
+3. **In Console** - You can also get notified in the Apify Console. This is useful if you are using if you are accessing the Apify Console often and you do not need to be notified as soon as possible.
 
-* [Add data validation](https://www.youtube.com/watch?v=Yha8X4cLtw8).
+![Notifications configurations](./images/notifications.png)
 
-We also have written tutorials for the following use cases.
+### Alert notification
 
-* [Monitor actor or task runs for failure](./actor_or_task_run_failure.md)
+The email and Slack alert notifications both contain the same information. You will receive a notification with the following information:
 
-* [Checking product-based data for correct format and duplicates](./check_data_quality.md)
+1. **Alert name**
+2. **Condition** - The condition that was violated.
+3. **Value** - The value of the metric violating the condition and triggering the alert.
+4. **Run ID** - The ID of the run that triggered the alert which links directly to the run detail in Apify Console.
+5. **Actor** - The full name of the actor that triggered the alert which links to the actor detail in Apify Console.
+6. **Task** - If the monitoring alert was set up for a task, then this field will contain the name of the task which links to the task detail in Apify Console.
 
-* [Monitoring shared datasets](./monitor_shared_datasets.md)
+![Email notification](./images/email-notification.png)
 
-* [Monitoring multiple actors or tasks](./monitor_multiple_tasks.md)
+While the in app notification will contain less information, it will point you directly to the actor or task that triggered the alert:
```

Comment:
```suggestion
While the in-app notification will contain less information, it will point you directly to the Actor or task that triggered the alert:
```